src/System.Buffers.Experimental/System/Buffers/Text/BufferWriter_writable.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text.Utf8;
+
+namespace System.Buffers.Text
+{
+    public ref partial struct BufferWriter
+    {
+        #region Byte
+        public bool TryWriteBytes(byte[] bytes)
```

Comment:
Do we need this API? Why can't we use implicit cast from array to ReadOnlySpan?

Reply:
For now I want to have this API as it came from WritableBufferWriter (with all its perf optimizations), but it's a good point about the overloads. I will measure perf and possibly remove/rename.